src/utils/forecastTemplates.js

```diff
@@ -1,4 +1,10 @@
-const forecastEmail = ({ city, temp_c, humidity, condition, unsubscribeLink }) => `
+const forecastEmail = ({
```

Comment:
За Clean Code краще використовувати action verbs в назві функцій (в цьому випадку, наприклад, buildForecastEmail)

```suggestion
const buildForecastEmail = ({
```